.github/oxid-esales/defaults/defaults.yml

```diff
@@ -274,6 +274,12 @@ install_module:
     artifact_prefix: 'configs-install_module'
   # Unique module id
   ids: &module_ids ''
+
+  # Title is used in the reports and debugging, if we install multiple modules
+  # debugging relies on sorting by titles, so prefixing it with numbers is
+  # recommended
+  title: *module_ids
```

Comment:
is it install_module_ids

Reply:
Not really. If you look at line 276, the anchor defined there is called "&module_ids" and I am referring to that anchor here the same way, I am referring to it in line 299